packages/cli-lib/lib/watch.js

```diff
@@ -33,10 +47,12 @@ function uploadFile(accountId, file, dest, options) {
 
   logger.debug('Attempting to upload file "%s" to "%s"', file, dest);
   const apiOptions = getFileMapperQueryValues(options);
+
   return queue.add(() => {
     return upload(accountId, file, dest, apiOptions)
       .then(() => {
         logger.log(`Uploaded file ${file} to ${dest}`);
+        notifyOfThemePreview(file, accountId);
```

Comment:
Is there a reason we're doing this in the watch library instead of the watch command? I ask because the original conception of the library package was that it could be used in other scenarios beyond just `@hubspot/cli`.

I also wonder if outputting as part of initial command output perhaps after the the uploading of the folder if `--initial-upload` is passed.